codes/quantum/qubits/CSS.yml

```diff
@@ -27,6 +27,7 @@ protection: 'Detects errors on \(d-1\) qubits, corrects errors on \(\left\lfloor
 
 notes:
```

Comment:
Add the homological discussion as a new paragraph in the code description, following Philippe's new formatting that allows for displayed eqs (if you want to have some): https://github.com/errorcorrectionzoo/eczoo_data/.